src/editor/utils/index.js

```diff
@@ -129,9 +130,12 @@ export const deepCopy = object => {
   return obj
 }
 
-export const loadImage = url => {
-  const image = new Image()
+export const loadImage = async (url, detectContentType) => {
+  if (detectContentType && await !checkImageContentType(url)) {
+    throw new Error('not a image')
```

Comment:
Is it too serious to throw an error directly? Maybe you need to return a rejected promise?

Reply:
Because of `Expected the Promise rejection reason to be an Error`. So `return Promise.reject` is redundant.